Dockerfiles/midstream/Dockerfile

```diff
@@ -14,7 +14,8 @@ RUN export ARCH=$(case $(arch) in x86_64) echo -n amd64 ;; aarch64) echo -n arm6
     mkdir /project/output && \
     chmod g+w /etc/passwd && \
     dnf install --setopt=install_weak_deps=False -y git-core ansible skopeo && \
-    dnf clean all
+    dnf clean all && \
+    git clone --branch $OPERATOR_TEST_PLAYBOOKS_TAG https://github.com/redhat-operator-ecosystem/operator-test-playbooks.git /project/operator-test-playbooks
```

Comment:
I would prefer to have `master` as a default instead of some ancient tag
relying on `v1.0.11` is not ideal.